content/docs/notebooks/troubleshoot.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "Troubleshooting Guide"
+description = "Fixing common problems in Kubeflow notebooks"
+weight = 50
++++
+
+## Persistent Volumes and Persistent Volumes Claims
+
+First, make sure that PVCs are bounded when using jupter notebooks. This should
+not be a problem when using managed Kuberenets. But if you are using Kubernetes
+on-prem, checkout [this](https://www.kubeflow.org/docs/use-cases/kubeflow-on-multinode-cluster/) guide.
```

Comment:
Use relative links for all links that point to other docs on our site. So instead of this:

```
https://www.kubeflow.org/docs/use-cases/kubeflow-on-multinode-cluster/
```

Use this:
```
docs/use-cases/kubeflow-on-multinode-cluster/
```

Relative links are essential for the docs versioning system, to allow people to remain in the same version of the docs rather than hopping out to the latest version when they click a link.